src/sinks/elasticsearch/encoder.rs

```diff
@@ -47,6 +47,12 @@ impl EventCount for ProcessedEvent {
     }
 }
 
+impl GetEventCountTags for ProcessedEvent {
+    fn get_tags(&self) -> vector_common::request_metadata::EventCountTags {
```

Comment:
You've got a fully qualified reference to `EventCountTags` even though you've imported it above.